### DISCUSSION TITLE
Anyone having success with the LORA implementation?

### DISCUSSION THREAD

**Discussion by andykaufseo** [d8ahazard/sd_dreambooth_extension#635] [Q&A]
I've tested it with this dreambooth extension (didn't test the official implementation) : it does what it says, fast speed (high batch count) and low vram usage (can get as low as 4.7gb). But the quality...

Regular dreambooth needs 100-200 steps per image to get it right (and maybe another 100-200 to get it perfect).
With the LORA thing checked, after 1000 steps per image it's not even close (i can see it's getting there, it has an idea of what's going on, but the results are like with DB after 20-50 steps). 

Tried different learning rates, captions / no captions, etc.

Anyone had any notable success with it?

@d8ahazard - everything implemented just like in the original paper? (i'm pretty sure it is, but i just wanted to double check with you).

**Comment by Thomas-MMJ** [comment]
What learning rate are you using?  should be 1e-4 to 1e-5 or so, not 1-e6 (typical for normal dreambooth).

Are you training the text encoder also or just the Unet?

**Comment by ghost** [comment]
Yeah, what are your parameters? I haven't tested objects, but i did get success with, let's say, states, or actions, and definitely styles. I even tried very small finue-tune with 620 images and it worked like a charm.

**Reply by andykaufseo** [reply]
how did you manage to train actions? what sort of captions and settings did you use? how many epochs did you train with those 620 images?

**Reply by ghost** [reply]
It's more of a state tbf, and model had rough idea of what it is, because it is a common enough tag, but never actually could show it, no matter how strong emphasis is. That specific training was not relevant to one with 620 images, 620 one is for general quality improvement, not to learn objects/actions.

To improve knowledge of state i just trained as it's recommended, ~100-125 steps per image, ~200 images about subject, and ~200 more from artists where it's not necesserily present, but sometimes is, nothing special. Captions are common, just smart tagger, though, with bigger size images, 768-1024, so it could tag better, and then mass deletion of them and replacement with uncropped ones for training purposes, since it's not relevant to crop images anymore? and i'd rather have full reso images scaled than cropped by tagger. And with my specific case i had to replace some of tags, because that specific state in very often treated as other.
